### ISSUE TITLE
add the simple file manager to Apollo

### ISSUE THREAD

**Issue by bucanero** [bucanero/apollo-ps3#43]
from @aldostools idea:
integrate the simple file manager into Apollo, so users can manually copy/move/delete files around within Apollo if needed

**Comment by retrosapien** [comment]
Adding to this, viewing the save files inside a virtual memory card and moving them between cards would be useful  too. (specifically for HEN users) 
=)

**Comment by aldostools** [comment]
> Adding to this, viewing the save files inside a virtual memory card and moving them between cards would be useful too. (specifically for HEN users)
> =)

Yes it would be useful, but it would more difficult to implement. Damian would need to create a hex viewer to do this.

The addition of the file manager more simple, because it is already encapsulated and he only needs to add libsfm_ps3.a, libfatfs.a and libntfs_ext.a to the project, then call the file manager using following code:
```
// exFAT File Manager
extern s32 fmapp_run();
fmapp_run();
```

**Comment by bucanero** [comment]
option to launch external File manager elf was added in [v2.0.2](https://github.com/bucanero/apollo-ps3/releases/tag/v2.0.2)
